src/python/pants/backend/go/goals/debug_goals.py

```diff
@@ -128,8 +137,8 @@ async def dump_go_import_paths_for_module(
             continue
 
         console.write_stdout(f"{tgt.address}:\n")
-        package_mapping = await Get(  # noqa: PNT30: requires triage
```

Comment:
looks like the auto migration stripped this comment, want me to add it back?

Reply:
https://github.com/pantsbuild/pants/issues/18365

That's the reason for it, and I guess the question is whether they can be written in parallel or not?

Reply:
Upon further look, no point in trying to sort this out right now - if it strips the tag, please put it back and we can triage it later

Reply:
done